scripts/CEE/pcap-collector/README.md

```diff
@@ -10,15 +10,19 @@ __NOTE:__ backplane cli >= 0.0.23 is needed.
 Parameters:
 - TIME: In seconds. Must be less than or equal to 900 (15 mins)
 - NODE: The node to run the packet capture on.
+- FILTERS: A valid set of PCAP filters for use in tcpdump
+- INTERFACE: The 
 
 ```bash
 # Capture traffic on node ip-10-0-253-170.ap-southeast-2.compute.internal for 10 mins
 ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal
 
+# Capture traffic for a particular interface in a node
+ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal -p INTERFACE=vxlan_sys_4789
```

Comment:
maybe update the example as `-p INTERFACE=all` as the vxlan is the default value anyway.